jose1xdd/Proyecto-Mobiles
# feat: Implement To-Do List using Jetpack Compose

# Pull Request Title

## Description

This PR introduces changes to the codebase to implement a To-Do list application using Jetpack Compose. It includes UI components such as TopAppBar, BottomAppBar, FloatingActionButton, and a list of tasks with checkboxes and delete functionality.

## Type of Change


## How Has This Been Tested?

I've tested the application by running it on different devices and screen sizes to ensure proper rendering and functionality. Instructions to reproduce the tests:
1. Run the application on an emulator or physical device.
2.. remove, and check tasks to verify task manipulation functionality.
3. Navigate through different screens to verify UI navigation.

## Checklist:

- [ ] I have checked that my change does not introduce any regressions in other modules

## Screenshots (if appropriate)

Include any screenshots that help visually explain your PR.

## Additional Comments
